region/lowernorfair/west/Fast Ripper Room.json

```diff
@@ -118,9 +118,49 @@
         "h_heatProof",
         "ScrewAttack",
         {"resetRoom": {"nodes": [1]}},
-        {"refill": ["Energy", "Super"]}
+        {"cycleFrames": 240},
+        {"or": [
+          "canInsaneJump",
+          {"and": [
+            "canFarmWhileShooting",
+            {"cycleFrames": 60}
+          ]},
+          {"cycleFrames": 360}
+        ]}
       ],
-      "resetsObstacles": ["A"]
+      "resetsObstacles": ["A"],
+      "farmCycleDrops": [
+        {"enemy": "Ripper 2 (red)", "count": 5}
+      ]
+    },
+    {
+      "link": [1, 1],
+      "name": "Partial Ripper Farm (Supers)",
+      "entranceCondition": {
+        "comeInNormally": {}
+      },
+      "requires": [
+        "ScrewAttack",
+        "canDodgeWhileShooting",
+        {"heatFrames": 200},
+        {"resourceAvailable": [{"type": "Energy", "count": 50}]},
+        {"partialRefill": {"type": "Super","limit": 5}},
+        {"heatFrames": 80}
+      ]
+    },
+    {
+      "link": [1, 1],
+      "name": "Partial Ripper Farm (Energy)",
```

Comment:
I'm not sure the intent here, but I'm interpreting this as a way to take advantage of health bomb to gain a bit of energy while not full on Supers? If full on Supers, the main farm-cycle strat could be applicable instead.

A pause abuse version could be added for this, similar to the "Approach Farm" strats that were used with the heated Gamet farms. I found it was fairly reasonable coming in with 15 energy: https://videos.maprando.com/video/7455 . Even without pause abuse, a strat like this can be helpful for modeling how you can get the farm started since the amount of heat damage until collecting your first drop is significantly less than the full cycle's worth of heat damage.

Reply:
It took some finessing...